app/src/main/res/layout/layout_quick_action_sheet.xml

```diff
@@ -22,109 +27,115 @@
+            android:textSize="12sp"
+            android:visibility="gone" />
 
-            <Button
-                android:id="@+id/quick_action_open_app_link"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:background="?selectableItemBackgroundBorderless"
-                android:drawableTop="@drawable/library_icon_app_links_circle_background"
-                android:drawablePadding="5dp"
-                android:text="@string/quick_action_open_app_link"
-                android:textAllCaps="false"
-                android:textColor="?primaryText"
-                android:textSize="12sp"
-                android:visibility="gone" />
+        <Button
+            android:id="@+id/quick_action_open_app_link"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:background="?selectableItemBackgroundBorderless"
+            android:drawableTop="@drawable/library_icon_app_links_circle_background"
+            android:drawablePadding="5dp"
+            android:text="@string/quick_action_open_app_link"
+            android:textAllCaps="false"
+            android:textColor="?primaryText"
+            android:textSize="12sp"
+            android:visibility="gone" />
 
-            <Button
-                android:id="@+id/quick_action_read"
-                android:layout_width="0dp"
-                android:layout_height="wrap_content"
-                android:layout_weight="1"
-                android:background="?selectableItemBackgroundBorderless"
-                android:drawableTop="@drawable/reader_two_state"
-                android:drawablePadding="5dp"
-                android:text="@string/quick_action_read"
-                android:textAllCaps="false"
-                android:textColor="?primaryText"
-                android:textSize="12sp"
-                android:visibility="gone" />
-        </LinearLayout>
+        <Button
+            android:id="@+id/quick_action_read"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:background="?selectableItemBackgroundBorderless"
+            android:drawableTop="@drawable/reader_two_state"
+            android:drawablePadding="5dp"
+            android:text="@string/quick_action_read"
+            android:textAllCaps="false"
+            android:textColor="?primaryText"
+            android:textSize="12sp"
+            android:visibility="gone" />
+    </LinearLayout>
 
-        <View
-            android:id="@+id/overlay"
-            android:layout_width="match_parent"
-            android:layout_height="match_parent"
-            android:alpha="0.0"
-            android:background="?foundation" />
-    </FrameLayout>
-</LinearLayout>
+    <View
```

Comment:
bringToFront() didn't work for me (maybe because it's a ConstraintLayout?), but I just made sure that it was drawn on top of `quick_action_buttons_layout` by using the buttons as the top constraint and having the overlay xml code below the buttons since android draws views starting at the top so it should work! Works on the S10